DonutsGo/DonutsGo.WebUI/Pages/Products/Products2.razor

```diff
@@ -0,0 +1,151 @@
+﻿
+
+
+@page "/waffles"
+@inject HttpClient Http
+
+
+ <div class="context"></div>
+
+
+        <div class="area" >
+            <ul class="circles">
+                    <li></li>
+                    <li></li>
+                    <li></li>
+                    <li></li>
+                    <li></li>
+            </ul>
+     
+
+    		
+	  	  
+ <nav id="main-nav">
+
+   <h1 class="logo">DonutsGo<br>
+ </h1>
+  
+  <ul>
+    <li><a class="active" href="#Log In">Log In</a></li>
+    <li><a href="home">Home</a></li>
+    <li><a href="">Products</a>
+      <ul>
+        <li><a href="products">Donuts</a></li>
+        <li><a href="waffles">Waffles</a></li>
+        <li><a href="#OSETROVNA">Drinks</a></li>
+      </ul>
+    </li>
+    <li><a href="#DOKUMENTY">Gift Ideas</a></li>
+  </ul>
+	</nav
+   <br>
+   <br>
+   <br>
+   <br>
+
+<h3>Waffles</h3>
+
+
+@if (products.Count==0)
+{
+    <h3>No products founds...</h3>
+}
+else
+{
+    <div class="d-flex  flex-wrap justify-content-between">
+        @foreach (var product in products)
+        {
+
+            <div class="card m-2" style="width: 35rem;">
+                <img class="card-img-top" src="http://donuterie.ro/img/donuts/Donuterie-Instalove.jpg" alt="Card image cap">
+                <div>
+                    <p>Name: @product.Name</p>
+                    <p>Price: @product.Price</p>
+                    <p>Type:@product.Type</p>
+                </div>
+            </div>
+
+        }
+    </div>
+
+
```

Comment:
For sure spacing in these razor files needs to be improved. It hurts my eyes when I try to read it. 🤣